packages/dashboard/src/components/palette/icons/index.css

```diff
@@ -1,18 +1,61 @@
 @import "../../../styles/variables.css";
 
 .palette-component-icon {
-  width: var(--size-component-palette-icon);
-  height: var(--size-component-palette-icon);
-  border: var(--border-width-component-palette-icon) solid;
-  border-color: var(--colors-grey);
-  border-radius: var(--radius-component-palette-icon);
-  color: var(--colors-black);
-  fill: var(--colors-black);
-  stroke: var(--colors-black);
+  position: relative;
   cursor: grab;
 }
 
-.palette-component-icon:hover,
-.palette-component-icon-active {
-  background-color: var(--colors-component-palette-active);
+.palette-component-icon .tooltiptext {
+  /* 14px is not provided in design token */
+  font-size: 14px;
+  position: absolute;
+  left: 50%;
+  top: 115%;
```

Comment:
When we design bottom tooltip then we need to provide top position, I have added 115% to manage arrow of tooltip. if we give 100% then it will look like this in the image below. 
![image](https://github.com/awslabs/iot-app-kit/assets/81667589/db6d0114-9374-4528-b4dd-d8c1b67f1482)